### ISSUE TITLE
[Usability] User experiences an unexpected interaction with a component.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80593]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User experiences an unexpected interaction with a component.
**VA.gov Experience Standard - category:** Usability
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the "Ask for your claim decision" page (eg. test user Kyle Cole/+23, [/your-claims/600494815/ask-va-to-decide](https://staging.va.gov/track-claims/your-claims/600494815/ask-va-to-decide)), the "Not yet--I still have more..." button appears to be purely navigational. It doesn't appear to save any user data, it just takes you back to your claim.

Maintaining material honesty around links and buttons (links navigate, buttons do things) helps provide users with predictable interactions --- especially assistive technology users, as some AT will have different interaction models for buttons and links. See the [Design System guidance on links and buttons](https://design.va.gov/components/link/#links-vs-buttons).

## Recommended action

Assuming that this is just navigational, this should probably be a link. Ideally I'd also like the text to make clear where it's going to take you, as right now it's not really clear where you're going to end up when you click the link.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by davidakennedy** [comment]
![Screenshot of Ask the VA to decide page with red arrow pointing to "Not yet–I still have more evidence to submit" button](https://github.com/department-of-veterans-affairs/va.gov-team/assets/1473618/ab667f7d-9f8a-4b48-9d08-0786de6ea50b)

Sharing a screenshot to help engineers here. This is a good change to make. It's interesting to me that clicking that button takes you back to the status page, not the files page, with the view details link. Unless I'm missing something, that's the only place you can get to the Ask VA to decide page, so maybe we take the Veteran back to the files page and send focus to the View details link.

**Comment by jacobworrell** [comment]
Feedback incorporated into designs for new 5103 request and submission experience. linking to epic and closing this ticket.

**Comment by jacobworrell** [comment]
re-opening added to blocked

**Comment by pmclaren19** [comment]
This ticket will be blocked until we release the https://github.com/department-of-veterans-affairs/va.gov-team/issues/81212 for Improving 5103 tracked items.

**Comment by pmclaren19** [comment]
This has been resolved.
